packages/browser/examples/svelte/README.md

```diff
@@ -0,0 +1,128 @@
+# Usage with Svelte
+
+Integration with Svelte is as simple as adding hooks (Server and Client) of handleError:
+
+- For handle error of server add handleError of HandleServerError type
+- For handle error of client add handleError of HandleClientError type
+
+## App configuration
+
+```ts
+// app.d.ts
+
+// Add interface of Error
+
+// See https://kit.svelte.dev/docs/types#app
+// for information about these interfaces
+// and what to do when importing types
+declare global {
+  namespace App {
+    interface Error {
+      message: unknown;
+      errorId: string;
+    }
+  }
+}
+
+export {};
+```
+
+## Server Hook
+
+To create an error handler for server with a `Notifier` initialized with your `projectId` and `projectKey`. In this example the handler will be in a file called `src/hooks.server.js`.
+
+```js
+// src/hooks.server.js
+import crypto from 'crypto';
+import { Notifier } from '@airbrake/browser';
+
+var airbrake = new Notifier({
+  projectId: 1, // Airbrake project id
+  projectKey: 'FIXME', // Airbrake project API key
+});
+
+airbrake.addFilter(function (notice) {
+  notice.context.hooks = 'server';
+  return notice;
+});
+
+/** @type {import('@sveltejs/kit').HandleServerError} */
+export function handleError({ error, event }) {
+  const errorId = crypto.randomUUID();
+  // example integration with https://airbrake.io/
+  airbrake.notify({
+    error: error,
+    params: { errorId: errorId, event: event },
+  });
+
+  return {
+    message: error,
+    errorId,
+  };
+}
+```
+
+## Client Hook
+
+To create an error handler for client with a `Notifier` initialized with your `projectId` and `projectKey`. In this example the handler will be in a file called `src/hooks.server.js`.
+
+```js
+// src/hooks.server.js
+import crypto from 'crypto';
+import { Notifier } from '@airbrake/browser';
+
+var airbrake = new Notifier({
+  projectId: 1, // Airbrake project id
+  projectKey: 'FIXME', // Airbrake project API key
+});
+
+airbrake.addFilter(function (notice) {
+  notice.context.hooks = 'client';
+  return notice;
+});
+
+/** @type {import('@sveltejs/kit').HandleClientError} */
+export function handleError({ error, event }) {
+  const errorId = crypto.randomUUID();
+  // example integration with https://airbrake.io/
+  airbrake.notify({
+    error: error,
+    params: { errorId: errorId, event: event },
+  });
+
+  return {
+    message: error,
+    errorId,
+  };
+}
+```
+
+## Test
+
+To test that server hook has been installed correctly in your Svelte project, just open up the JavaScript console in your internet browser and paste in:
```

Comment:
Oh, you could use imports inside dev tools? I had no idea 😅

Reply:
:stuck_out_tongue: